Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python as base
+
+# Perform common operations, dependency installation etc...
+RUN pip install poetry
+RUN pip install flask
```

Comment:
We should be able to remove this line - pip is useful installing `poetry`, but we then want to offload our dependency management onto Poetry, so we don't need to explicitly add any libraries, because the `poetry install` below will handle that for us
```suggestion
```